test/numerical/TestGemm.cpp

```diff
@@ -10,13 +10,12 @@
 #include <string>
 #include <vector>
 
-#include "mlir/IR/BuiltinOps.h"
 #include "llvm/Support/FileSystem.h"
 
 #include "src/Compiler/CompilerUtils.hpp"
-#include "src/Dialect/ONNX/ONNXOps.hpp"
 #include "src/Runtime/ExecutionSession.hpp"
 #include "src/Runtime/OMTensorHelper.h"
+#include "test/modellib/ModelLib.hpp"
 
 #define SHARED_LIB_BASE string("./TestGemm_main_graph")
```

Comment:
[Suggestion]:  
``` 
`#define SHARED_LIB_BASE string("./TestGemm_main_graph")` --> static const llvm::StringRef("./TestGemm_main_graph");`

Reply:
This could even be computed based on the file name. I considered doing that when I made the tests work for Windows, but I decided against it at the time to keep the change as small as possible.

Reply:
change the code to use the static const construct